package/cfg/k3s-cis-1.9/policies.yaml

```diff
@@ -45,48 +45,50 @@ groups:
             fi;
           done
 
+          cr_whitelist="cluster-admin k3s-cloud-controller-manager local-path-provisioner-role"
+          cr_whitelist="$cr_whitelist system:kube-controller-manager system:kubelet-api-admin system:controller:namespace-controller"
+          cr_whitelist="$cr_whitelist system:controller:disruption-controller system:controller:generic-garbage-collector"
+          cr_whitelist="$cr_whitelist system:controller:horizontal-pod-autoscaler system:controller:resourcequota-controller"
           # Check ClusterRoles
           kubectl get clusterroles -o custom-columns=CLUSTERROLE_NAME:.metadata.name --no-headers | while read -r clusterrole_name
           do
             clusterrole_rules=$(kubectl get clusterrole "${clusterrole_name}" -o=json | jq -c '.rules')
-            if echo "${clusterrole_rules}" | grep -q "\[\"\*\"\]"; then
-              echo "**clusterrole_name: ${clusterrole_name} clusterrole_rules: ${clusterrole_rules} clusterrole_is_compliant: false"
+            if echo "${cr_whitelist}" | grep -q "${clusterrole_name}"; then
+              printf "**clusterrole_name: %-50s is_whitelist: true  is_compliant: true\n" "${clusterrole_name}"
+            elif echo "${clusterrole_rules}" | grep -q "\[\"\*\"\]"; then
+              echo "**clusterrole_name: ${clusterrole_name} clusterrole_rules: ${clusterrole_rules} is_compliant: false"
             else
-              printf "**clusterrole_name: %-50s clusterrole_is_compliant: true\n" "${clusterrole_name}"
+              printf "**clusterrole_name: %-50s is_whitelist: false is_compliant: true\n" "${clusterrole_name}"
             fi;
           done
         use_multiple_values: true
         tests:
-          bin_op: or
           test_items:
-            - flag: "role_is_compliant"
-              compare:
-                op: eq
-                value: true
-              set: true
-            - flag: "clusterrole_is_compliant"
+            - flag: "is_compliant"
               compare:
                 op: eq
                 value: true
-              set: true
         remediation: |
           Where possible replace any use of wildcards in clusterroles and roles with specific objects or actions.
-          K3s requires exceptions for following cluster roles, whic are required for regular operations:
-          - k3s-cloud-controller-manager, local-path-provisioner-role
-        scored: false
+          K3s gives exceptions for following cluster roles, which are required for regular operations:
+          - k3s-cloud-controller-manager, local-path-provisioner-role, cluster-admin
+          - system:kube-controller-manager, system:kubelet-api-admin, system:controller:namespace-controller, 
+          - system:controller:disruption-controller, system:controller:generic-garbage-collector, 
+          - system:controller:horizontal-pod-autoscaler, system:controller:resourcequota-controller
```

Comment:
Do we want to specify the nature (what are their usage) and justification for each cluster role that are whitelisted ? Maybe this could be later documented into the K3s Docs and a link to it can be added there.